src/app/core/collection.service.ts

```diff
@@ -242,7 +243,8 @@ export class CollectionService {
           {
             email,
             name
-          }
+          },
+          { headers: this.headers, withCredentials: true }
```

Comment:
Where are headers initialized? Will it automatically put the bearer token?

Reply:
this.headers does include that in the services on the client

Reply:
Up on line 22 we declare our Headers